features/progress_bar.feature

```diff
@@ -0,0 +1,29 @@
+@javascript @seed_topics
+Feature: Allow user to keep track of their survey completion status by using a progress bar
+    As a new user filling out the survey
+    So that I can know how close I am to finishing
+    I want to see my progress
+
+Background: I am a new user
+    Given the following questions exist:
+        | text                          | topic           | index |
+        | Opinion on climate changes?   | Climate         | 1     |
+    And the following responses exist:
+        | question_text                 | response_text                                     | index |
+        | Opinion on climate changes?   | Climate change is happening.                      | 1     |
+    And I have selected the topics "Education", "Climate", "Philosophy", "Technology", "Religion"
+
+Scenario: Progress advances when a question is completed
+    Given I am on topic ID 1
```

Comment:
Much prefer instead the step:

Given I am on the topic "Education". Then use the database to find the ID.